docs/about.md

```diff
@@ -0,0 +1,73 @@
+
+The *backup* operation (1) uploads a tarball of copied Kubernetes resources into cloud object storage and (2) uses the cloud provider API to make disk snapshots of persistent volumes, if specified.
+
+You can optionally specify hooks that should be executed during the backup. For example, you may
+need to tell a database to flush its in-memory buffers to disk prior to taking a snapshot. You can
+find more information about hooks [here][10].
+
+Some things to be aware of:
+* *Cluster backups are not strictly atomic.* If API objects are being created or edited at the time of backup, they may or not be included in the backup. In practice, backups happen very quickly and so the odds of capturing inconsistent information are low, but still possible.
+
+* *A backup usually takes no more than a few seconds.* The snapshotting process for persistent volumes is asynchronous, so the runtime of the `ark backup` command isn't dependent on disk size.
+
+These ad-hoc backups are saved with the `<BACKUP NAME>` specified during creation.
+
+### Schedules
+
+The *schedule* operation allows you to back up your data at recurring intervals. The first backup is performed when the schedule is first created, and subsequent backups happen at the schedule's specified interval. These intervals are specified by a Cron expression.
+
+A Schedule acts as a wrapper for Backups; when triggered, it creates them behind the scenes.
+
+Scheduled backups are saved with the name `<SCHEDULE NAME>-<TIMESTAMP>`, where `<TIMESTAMP>` is formatted as *YYYYMMDDhhmmss*.
+
+### Restores
+
+The *restore* operation allows you to restore all of the objects and persistent volumes from a previously created Backup. Heptio Ark supports multiple namespace remapping--for example, in a single restore, objects in namespace "abc" can be recreated under namespace "def", and the ones in "123" under "456".
+
+Kubernetes API objects that have been restored can be identified with a label that looks like `ark-restore=<BACKUP NAME>-<TIMESTAMP>`, where `<TIMESTAMP>` is formatted as *YYYYMMDDhhmmss*.
+
+You can also run the Ark server in *restore-only* mode, which disables backup, schedule, and garbage collection functionality during disaster recovery.
+
+## Backup workflow
+
+Here's what happens when you run `ark backup create test-backup`:
+
+1. The Ark client makes a call to the Kubernetes API server to create a `Backup` object from the custom resource. (TODO: check resource/object language).
+
+1. The `BackupController` notices the new `Backup` object and performs validation.
+
+1. The `BackupController` begins the backup process. It collects the data to back up by querying the API server for resources. (oy, overloaded term?)
+
+1. The `BackupController` uploads the backup file makes a call to the object storage service -- for example, AWS S3.
+
+Note that by default Ark also makes disk snapshots of any persistent volumes. The snapshots can be disabled with the option `--snapshot-volumes=false`.
```

Comment:
It's a bit more subtle than "any persistent volumes". If you just run `ark backup create`, it will do it for all of them. If you specify `--include-namespaces`, it will only do it for PVs that are associated with PVCs in the selected namespaces. Similar for the label selector as well.